README.md

```diff
@@ -0,0 +1,22 @@
+# Welcome to the scraper!
+
+This scraper current runs on bitcoin stackexchange, the bitcoin optech repo, the bitcoin talk forum and bitcoin transcripts.
+
+## Setup
+
+You need an env file to indicate where you are pushing the data.
+
+1. Copy `cp .env.sample .env`
+2. run `yarn install && cd common && yarn install && cd ..`
```

Comment:
Currently, we have to run `yarn install` for each scraper.
I believe it is more like this:
```suggestion
2. run `cd common && yarn install && cd ../mailing-list && yarn install && cd ..`
```

Maybe we should create a single package.json for all the packages because most scrapers require the same packages.
This would make the package installation much easier.